### PR TITLE
build 4.14+trunk with dune.2.9.0

### LINKED ISSUE #247: Sandmark breaks on OCaml 4.14.0+trunk
See the failure in the CI at https://cloud.drone.io/ocaml-bench/sandmark/840/1/2. It would be useful to keep the CI green in the next few months as we head towards upstreaming multicore.

### PR DESCRIPTION
**Pull request by shubhamkumar13**
The PR aims to resolve #247.  

The packages that are still failing to build on this PR for 4.14.0+trunk
```
react
rresult
fmt
astring
mtime
ocaml-migrate-parsetree
```

### CODE REVIEW

File: run_config.json [hunk 1]
```diff
@@ -18,26 +18,6 @@
     }
   ],
   "benchmarks": [
-    {
```
Comment (kayceesrk) [inline run_config.json:21]: Why are these deleted? If I merge this, then these benchmarks will not work on 4.12+stock, isn't it?

Comment (shubhamkumar13) [reply run_config.json]: I deleted them to remove the benchmarks from being run. I also deleted the benchmarks associated with them
That was a wrong decision on my part. I have reverted the last commit.
To make sure the benchmarks that don't build because of https://github.com/ocaml-bench/sandmark/issues/250 I have commented out their alias instead of removing them.

File: benchmarks/irmin/dune [hunk 2]
```diff
@@ -8,4 +8,4 @@
 	(modules layers)
 	(libraries ppx_deriving_yojson.runtime irmin-pack irmin-pack.layered irmin-layers index fmt logs lwt lwt.unix yojson irmin unix cmdliner))
 
-(alias (name buildbench) (deps layers.exe irmin_mem_rw.exe))
+;;(alias (name buildbench) (deps layers.exe irmin_mem_rw.exe))
```
Comment (kayceesrk) [inline benchmarks/irmin/dune:11]: As we discussed, `;;` should be removed.

File: benchmarks/valet/dune [hunk 3]
```diff
@@ -4,4 +4,4 @@
  (name test_lwt)
  (libraries uuidm ocplib-endian unix react lwt lwt.unix))
 
-(alias (name buildbench) (deps test_lwt.exe))
\ No newline at end of file
+;;(alias (name buildbench) (deps test_lwt.exe))
```
Comment (kayceesrk) [inline benchmarks/valet/dune:7]: As we discussed, `;;` should be removed.

File: run_all_serial.sh [hunk 4]
```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-TAG='"macro_bench"' make run_config_filtered.json
```
Comment (kayceesrk) [inline run_all_serial.sh:3]: Don't change this file.

File: run_config.json [hunk 5]
```diff
@@ -491,8 +491,7 @@
       "executable": "benchmarks/valet/test_lwt.exe",
       "name": "test_lwt",
       "tags": [
-        "10s_100s",
```
Comment (kayceesrk) [inline run_config.json:494]: The changes to this file should be reverted.

**Comment by kayceesrk** [comment]
This PR should also update the README.md file to indicate that `dune.2.9.0` should be installed.

**Comment by shubhamkumar13** [comment]
This PR was superseded by https://github.com/ocaml-bench/sandmark/pull/251.

And is no longer needed as the above is merged.
